department-of-veterans-affairs/caseflow
# Queue | link to hearing worksheet print view

Connects #4816

### Description
Links to Hearing Worksheet print view

### Acceptance Criteria 
- [ ] 'View Hearing Worksheet' link in `AppealDetail` opens `/hearings/${hearing.id}/worksheet/print` in a new tab

### Testing Plan
1. Go to detail view of appeal with a non-cancelled hearing
2. Click 'View Hearing Worksheet', verify you are directed to print the worksheet
